src/CommitmentsV2/SFA.DAS.CommitmentsV2.ExternalHandlers/Settings.job

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
Is this file necessary if the entire contents are commented out?

Reply:
Done